tests/test_migration_check.py

```diff
@@ -0,0 +1,23 @@
+from incuna_test_utils.compat import DJANGO_LT_17
+
+if not DJANGO_LT_17:
+    from django.core.management import call_command, CommandError
+    from django.test import modify_settings, TestCase
```

Comment:
`modify_settings` is new in `1.7`, so it needs to be here and we can't use a `skipIf` instead. I put everything in the `if` for simplicity.